api_client/python/timesketch_api_client/timeline.py

```diff
@@ -277,16 +276,19 @@ def run_analyzers(
                 'No session data returned back, analyzer may have run but '
                 'unable to verify, please verify manually.')
 
-        session_id = objects[0].get('analysis_session')
-        if not session_id:
+        analyzer_results = []
+        for session in objects:
+            analyzer_result = analyzer.AnalyzerResult(
+                timeline_id=self.id, session_id=session.id,
+                sketch_id=self._sketch_id, api=self.api)
+            analyzer_results.append(analyzer_result)
+
+        if not analyzer_results:
             raise error.UnableToRunAnalyzer(
-                'Analyzer may have run, but there is no session ID to '
+                'Analyzer may have run, but there is no sessions to '
                 'verify that it has. Please verify manually.')
 
-        session = analyzer.AnalyzerResult(
-            timeline_id=self.id, session_id=session_id,
-            sketch_id=self._sketch_id, api=self.api)
-        return session
+        return analyzer_results
```

Comment:
can you change the docstring as well, the returns section still says a single analyzer result is returned

Reply:
Done